api/PerformanceNavigation.json

```
@@ -169,10 +169,10 @@
               "version_added": null
             },
             "firefox": {
-              "version_added": "25"
+              "version_added": false
```

I'm suggesting that we do something similar to the change for `type` here:

```suggestion
              "version_added": "25",
              "notes": "This property can return incorrect <code>type</code> values. See <a href='https://bugzil.la/1459711'>bug 1459711</a>."
```